Check Accessiblity

Yemisirach opened the pull request:
### Project requirements

- We checked all of the items listed in the article.
- We determine if Portfolio needs any changes.
- We make changes in a separate branch and open a new pull request.
- We prepare a report listing all points that we have checked, and the result of our evaluation.
- Our report included ; 
 
- [Page titles](https://www.w3.org/WAI/test-evaluate/preliminary/#title) - 
- [X]  page title-:  Correct misspelling
- [Image text alternatives** (`<img alt="alt text">`)](https://www.w3.org/WAI/test-evaluate/preliminary/#images)
- [X]  Image text alternative-:  no issues found
 - [Text headings](https://www.w3.org/WAI/test-evaluate/prelidminary/#headings)
 - [X] Text headings -:  no issues found
 - [Color contrast](https://www.w3.org/WAI/test-evaluate/preliminary/#contrast)
  - [X] Color contras -:  Added the color #7F8CFF for a text LET'S CONNECT.
  - [Resize](https://www.w3.org/WAI/test-evaluate/preliminary/#resize)
  - [X] Resize -:  Added the size for the text LET'S CONNECT.
  - [Interaction](https://www.w3.org/WAI/test-evaluate/preliminary/#interaction) 
  - [X] Interaction -:  no found on the  Figma design. 
  - [Moving content](https://www.w3.org/WAI/test-evaluate/preliminary/#moving)
  - [X] Moving content -:  no found on the  Figma design
  - [Multimedia](https://www.w3.org/WAI/test-evaluate/preliminary/#media)
   - [X] Multimedia -: no found on the  Figma design
   - [The basic structure of the page](https://www.w3.org/WAI/test-evaluate/preliminary/#structure)
   - [X] The basic structure of the page -:  no issues found

Our Group ; 
@JansueT 
@Kwaku28
@Yemisirach

iambenkis commented on index.html:
```diff
@@ -1,301 +1,217 @@
+        <section class="About">
+            <div class="ellipse">
+                <img class="ellipse_zero" src="./image/Ellipse 8.png" alt="ellipse">
+                <div class="ellipse_1">
+                    <img class="ellipse_zero_1" src="./image/Ellipse 5.png">
+                    <div class="ellipse_2">
```
- [ ] You add the alternatives on your images almost everywhere :clap: , but I found that you forget to add on this image :squid: , Kindly make sure you add it here.
`<img class="ellipse_zero_1" src="./image/Ellipse 5.png" alt="image_alt">`

iambenkis commented on index.html:
```diff
@@ -1,301 +1,217 @@
+<li class="language_list"><img src="./image/Ellipse 1 (1).png" alt="html"><h4>HTML</h4>
+</li>
+<li class="language_list"><img src="./image/Ellipse 1 (2).png" alt="css"><h4>CSS</h4>
+</li></ul></div></div></div><div class="framework Framework"><h2>Frameworks</h2><a href="#"><img src="./image/Union.png" alt="Union"></a></div><div class="framework"><h2>Skill</h2><a href="#"><img src="./image/Union.png" alt="Union"></a></div></div></section><!-- About section end --><section class="contact_me_section"><div class="Contact_me"><header class="contact_header"><h2>Contact me</h2></he ...
+              feature that you need built or a project that need coding. I'd
```
- [ ] Instead of using `h2` here :blush: Please use `h1` because this is the main heading of your page and according to the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :+1:
`<h1>Contact me</h1>`

iambenkis reviewed (changes requested):
# Required Changes :recycle:
![almost-there](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQp9z2Rq_bAJ42eRNxObBr1nKDUbZjLw2QqkQ&usqp=CAU)

### Hey Team (@Yemisirach , @KALUNGI-J-BOBSON ) :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Basic Structure is good
:heavy_check_mark: Can be maximized up to 200%


Good job on implementing some requirements, but there are some issues that you still need to work on to go to the next project, but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question, so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Yemisirach commented:
# Changes Request  ♻️
- [Image text alternatives** (`<img alt="alt text">`)](https://www.w3.org/WAI/test-evaluate/preliminary/#images)
- [X]  Image text alternative-:  Added alt on index.html on line 35
 - [Text headings](https://www.w3.org/WAI/test-evaluate/prelidminary/#headings)
 - [X] Text headings -: Added h1 heading instead of h2 heading in contact me section

ha-manel reviewed (approved):
# STATUS: APPROVED ✅

## Hi Team👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

<img src="https://media3.giphy.com/media/l4JySAWfMaY7w88sU/giphy.gif"/>

## Highlights:
* Well-written PR description ✔️ 
* All tests passed ✔️ 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
